src/GraphLayout.jl

```diff
@@ -2,11 +2,30 @@ __precompile__(true)
 
 module GraphLayout
     using Compose   # for plotting features
+    using GeometryTypes   # for generating the layout
 
     typealias AdjList{T} Vector{Vector{T}}
 
+    type Node
+      p::Point
+    end
+
+    type Edge
+      src::Point
+      dest::Point
+      directed::Bool
+    end
+
+    immutable Network
+      Positions::Array{Node}
+      Connections::Array{Edge}
```

Comment:
The Edge type duplicates the Nodes... This should rather be an index list into Positions